contributing/submitting-pr.md

```diff
@@ -90,6 +90,47 @@ Keep in mind this guidelines when writing changelog messages:
 
 ## Testing
 
+Testing our codes ensures that it's working as expected, so writing tests is an essential part of the contribution process. If you're contributing with a new feature, you'll have to write new tests from scratch. On the other hand, if you're fixing a bug, you might have to modify existing tests and add new tests to ensure that the bug has been fixed.
+
+Ensuring that your new feature works is just as important as ensuring that we're helping the user when things don't work as expected. For example, say you're working on a function named `add` that adds two numbers:
+
+```python
+def add(x, y):
+    return x + y
+```
+
+A test might look like this:
+
+```python
+def test_add():
+    assert add(21, 21) == 42
+```
+
+However, you should also take into account scenarios where the function doesn't work,
+for example, if the user calls the function with a string: `add("21", 21)`, they'll
+see the following error:
+
+```
+TypeError: can only concatenate str (not "int") to str
+```
+
+The error is difficult to understand, so you should ensure that you're covering likely
+scenarios (they don't have to be exhaustive but try to think what the user might do).
+After encountering such edge case, you can add a new test:
+
+
+```python
+def test_error_if_wrong_type():
+    with pytest.raises(TypeError) as excinfo:
+        add(21, 21)
+
+    message = "function add only supports numbers, you passed a str object in the first argument ('21')"
+    assert str(excinfo).value == message
+```
+
+Of course, the test won't pass since you haven't implemetned such validation, but you can now go, implement it and verify that the test passes.
```

Comment:
`implemetned` - typo

Reply:
fixed

Reply:
I still see `implemetned`

Reply:
maybe github  doesn't update the view? this is the commit that I added, you can see I fixed the typo: https://github.com/ploomber/contributing/pull/82/commits/a6895752190de3974ee6cf74d88f1f7ae1de749b